src/rebar.app.src

```diff
@@ -15,6 +15,11 @@
                   tools,
                   eunit,
                   common_test,
+                  dialyzer,
```

Comment:
dialyzer has `wx` in `applications`, will there be cases where `wx` is not built?